finesse/FINESSE_USAGE.md

```diff
@@ -0,0 +1,82 @@
+# Design of the Finesse Benchmark Tool
```

Comment:
we want to generalize to backend API testing, not just Finesse. review documentation to update.

Reply:
I've reviewed the documentation, and I believe it's fine as it is. The markdown file explains how the Finesse locust script works, and the README.md covers backend API testing, redirecting users to specific locust scripts for different purposes.